recipes/aravis/all/patches/0.8.25-gst-shared-lib.patch

```diff
@@ -0,0 +1,11 @@
+--- a/gst/meson.build	2023-10-03 14:55:30.481822400 -0700
++++ b/gst/meson.build	2023-10-03 14:56:40.939480100 -0700
+@@ -17,7 +17,7 @@
+ 
+ gst_plugin_filename = 'gstaravis.@0@'.format (aravis_api_version)
+ 
+-gst_plugin = shared_library (gst_plugin_filename,
++gst_plugin = library (gst_plugin_filename,
```

Comment:
Can I get some context into this patch @fixstars-sonali? Thanks!

Reply:
this allows us to find the gst plugins whether they're built as shared or static. Most people on a "normal" system would have them shared but of course in conan everything is static by default, that's my guess